src/pages/get-started/index.tsx

```diff
@@ -15,8 +15,8 @@ export const MostPopularBadge: FunctionComponent = () => (
     </span>
 )
 
-export const BestForTitle: FunctionComponent = () => (
-    <div className={`text-uppercase font-weight-bolder mb-1 ${styles.bestForTitle}`}>Best For</div>
+export const DesignedForTitle: FunctionComponent = () => (
+    <div className={`text-uppercase font-weight-bolder mb-1 ${styles.designedForTitle}`}>Designed For</div>
```

Comment:
@bretthayes Highlighting that the update from "Best For" -> "Designed For" also resulted in the name change to `DesignedForTitle`. ef9ef01